app/javascript/components/recipes/new/container.vue

```diff
@@ -9,6 +9,7 @@
       >
         <div class="justify-self-start">
           <input
+            v-model="element.ingredientQuantity"
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected mutation of "element" prop  vue/no-mutating-props